### PR TITLE
fix: wrong link to fsspec

### PR DESCRIPTION
**Pull request by deep-diver**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # `hffs`
 
-`hffs` builds on [`huggingface_hub`](https://github.com/huggingface/huggingface_hub) and [`fsspec`](https://github.com/huggingface/huggingface_hub) to provide a convenient Python filesystem interface to 🤗 Hub.
+`hffs` builds on [`huggingface_hub`](https://github.com/huggingface/huggingface_hub) and [`fsspec`](https://filesystem-spec.readthedocs.io/en/latest/) to provide a convenient Python filesystem interface to 🤗 Hub.
```
Comment (albertvillanova) [inline README.md:3]: Maybe using the GitHub link instead, to be consistent with the link just before?
```suggestion
`hffs` builds on [`huggingface_hub`](https://github.com/huggingface/huggingface_hub) and [`fsspec`](https://github.com/fsspec/filesystem_spec) to provide a convenient Python filesystem interface to 🤗 Hub.
```

**Review by albertvillanova — APPROVED** [review]
@deep-diver, thanks for pointing out the wrong link.

File: README.md (hunk 1 above)
Comment (deep-diver) [reply README.md]: Sounds better :)

**Review by mariosasko — APPROVED** [review]
Yes, let's keep GH links. Thanks!

**Pull request merged**